File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.java
class Solution {
    public int[] solution(int[] numbers, int num1, int num2) {
        int[] answer = new int[num2 - num1 + 1];
        
        for(int i = 0; i < numbers.length; i++) {
            if(num1 <= i && i <= num2) {
                answer[i - num1] = numbers[i];
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.go
func solution(numbers []int, num1 int, num2 int) []int {
    answer := make([]int, num2 - num1 + 1)
        
    for i := 0; i < len(numbers); i++ {
        if num1 <= i && i <= num2 {
            answer[i - num1] = numbers[i]
        }
    }

    return answer
}